cfgov/search/elasticsearch_helpers.py

```diff
@@ -72,10 +72,11 @@ def get_synonyms():
 
 
 def environment_specific_index(base_name):
-    if settings.DEPLOY_ENVIRONMENT in (None, '', 'local', 'production'):
+    env = settings.DEPLOY_ENVIRONMENT
```

Comment:
Why don't we call `lower` once here?
```suggestion
    env = settings.DEPLOY_ENVIRONMENT.lower()
```

Reply:
can't call .lower() on None, if that happens.

Reply:
Ah, I see.